### PR TITLE
Support for TensorFlow + Python 3

### PR DESCRIPTION
**Pull request by polzounov**
This is a very clean implementation of DKT, so I thought I would add TensorFlow and Python 3 support (may be useful to some people). 
I also updated `model.add(TimeDistributedDense(...)))` to `model.add(TimeDistributed(Dense(...)))`, to match newer versions of Keras.

Thanks for putting up the code!

### CODE REVIEW

**Comment by wunianjian** [comment]
Thanks for your sharing! But when I use your code, why is the loss much higher than the original and the AUC is only about 0.61?

**Comment by polzounov** [comment]
Did you try using both the original and this version?

**Comment by wunianjian** [comment]
Yes, the original version works well with 0.86 AUC.

**Comment by polzounov** [comment]
Not too sure, have you only ran each experiment once?
